src/backend/controllers/userController.js

```diff
@@ -75,6 +76,33 @@ exports.register = async (req: $Request, res: $Response) => {
 			}
 		});
 
+		// Creates an initial tank for the user
+		user = User.findOne({userName: userName, email: email}, (err: Error) => {
+			if (err) {
+				console.error(err.message);
+				return res
+					.status(500)
+					.json({ msg: 'Could not load created user' });
+			}
+		});
+
+		const initTank = new Tank({
```

Comment:
Did you check opening up Casus for this? I think you might need to have a container block with an empty block inside as the default casus block for things to work correctly

Reply:
Oh do I? Alright. I'll double check. Mind putting what the blank container will look like in a comment below me?

Reply:
I think this is what he is referring to.

function getTestTank(id: number=1): Tank {
	const position=id===1?new Vec(20, -20):new Vec(50, 40);
	const toReturn: Tank = new Tank(
		position,
		new Chassis(id===1?'CHASSIS_5':'CHASSIS_2'), 
		new Treads(id===1?'TREAD_2':'TREAD_1'), 
		new Gun(id===1?'GUN_3':'GUN_4', false),
		new Gun(id===1?'GUN_9':'GUN_10', true),
		new Scanner(false, false, id===1?'MEDIUM':'SMALL'),
		new Jammer(id===1?'MEDIUM':'LARGE'),
		getEmptyCasusCode()
	);
	loadCasus(blocks => {toReturn.casusCode = blocks});
	return toReturn;
}

function getEmptyCasusCode(): CasusBlock {
	const childrenBlocks: Array<CasusBlock> = [];
	const container: ContainerBlock = new ContainerBlock(childrenBlocks);
	return container;
}

Reply:
Lol it wouldn't print it in code view. Check TankLoader.js in master.

Reply:
That might work? You would need to save the stringified version of that probably since this is after is is saved, but yeah, that's pretty much what it would look like. You can just use the default constructor (you don't need to pass in an empty array).

Reply:
I know what to do.  I'll just use the code from SittingDuck. Just have to export the tanks and copy the casusCode structure from the JSON file